proxy/Dockerfile

```diff
@@ -55,4 +55,4 @@ EXPOSE 8080
 USER nginx
 STOPSIGNAL SIGTERM
 RUN nginx -V
-CMD ["nginx", "-e", "/home/nginx/log/error.log", "-g","daemon off;"]
\ No newline at end of file
+CMD ["nginx", "-e", "/home/nginx/log/error.log", "-g","daemon off", "--with-http_realip_module"]
```

Comment:
very interesting.
what does this module do?

Reply:
I saw it as a way to get the real IP from the many stackoverflow questions we were searching through but I am going to remove it on dev to see if we actually need it 🤔 

https://nginx.org/en/docs/http/ngx_http_realip_module.html
https://serverfault.com/questions/314574/nginx-real-ip-header-and-x-forwarded-for-seems-wrong

Reply:
that and also the stuff we added in the ngnix.conf

Reply:
Good catch, Aurora! It was not needed so it has been removed :D